cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperator.java

```diff
@@ -2247,7 +2314,7 @@ String getCertificateThumbprint(Secret certSecret, CertAndKeySecretSource custom
             }
         }
 
-        ConfigMap getKafkaAncialiaryCm()    {
+        ConfigMap getKafkaAncillaryCm()    {
```

Comment:
Could you fix the name also for other `ancialiary`? Can be in separate PR, but having each of them different is weird.